### ISSUE TITLE
Fix Download Links

### ISSUE THREAD

**Issue by WilsonGoins** [ufosc/BytesOfLoveWebsite#15]
The buttons to download the game currently only work on Windows. All three links are the same, so if Mac or Linux users try to download the game it will say it is not compatible with their system. 

So the links for Mac and Linux download buttons need to be fixed. Changes to the code need to be made wherever these buttons appear which is in Home.js and Download.js

**Comment by vule19** [comment]
Can I work on this issue?

**Comment by WilsonGoins** [comment]
> Can I work on this issue?

Yes! Ren’Py has functionality to create these folders which should be stored in a google drive and the link allows you to download it. I don’t know the specifics, if you need help reach out on the Open Source Club discord in the “bytes-of-love” or “bytes-of-website” channels.

**Comment by vule19** [comment]
Hi! i wanted to change the issue I want to fix, hope that okay.

**Comment by WilsonGoins** [comment]
> Hi! i wanted to change the issue I want to fix, hope that okay.

Yep no problem

**Comment by jonana-split** [comment]
hello! could I be assigned to this issue?

**Comment by WilsonGoins** [comment]
> hello! could I be assigned to this issue?

Sorry for the late response, sure! Let me know if you have any trouble

**Comment by zeim839** [comment]
Suggestion:
1. Compile the game for MacOS, Windows, Linux.
2. Upload binaries to [GitHub releases](https://github.com/ufosc/VisualNovel/releases)
3. Add an href to the Bytes of Love website that links to the GitHub releases page.
4. Users now have access to the full release history (old versions, etc.) and you don't have to update the site every time the binary changes or a new version is released.

**Comment by pranavrevuri** [comment]
Hello, I would like to work on this issue for my CEN3031 course. Please let me know if it is possible

**Comment by WilsonGoins** [comment]
> Hello, I would like to work on this issue for my CEN3031 course. Please let me know if it is possible
> 
> 

Yes, go ahead.
